rosbag2_cpp/CMakeLists.txt

```diff
@@ -25,6 +25,16 @@ option(DISABLE_SANITIZERS "disables the use of gcc saniztizers")
 # https://github.com/google/sanitizers/issues/794
 if(NOT ${CMAKE_SIZEOF_VOID_P} EQUAL 8)
   set(DISABLE_SANITIZERS ON)
+elseif(NOT DISABLE_SANITIZERS)
```

Comment:
can be logic below also be used on 32bit processors? That's the reason that `if` is there in the first place, because 32bit ARM doesn't have the sanitizers. I think I would prefer a `one-to-rule-them-all` logic.

Reply:
Actually, I think it can. The same error that I'm seeing on Fedora (missing `-llsan`) is reported in the linked GitHub issue. I'll update it.